styles/Elastic/WordChoice.yml

```diff
@@ -1,31 +1,46 @@
 extends: substitution
-message: "Consider using '%s' instead of '%s'."
+message: "Consider using '%s' instead of '%s', unless the term is in the UI."
 link: https://docs.elastic.dev/tech-writing-guidelines/word-choice
 ignorecase: true
 level: suggestion
 action:
   name: replace
 swap:
+  abnormal: "atypical, not typical"
   abort: "stop, cancel, end"
+  abuse: "misuse"
   blacklist: "blocklist"
   boot: "start, run"
   choose: "select"
   clone: "copy"
   could: "can, might"
   delete: "remove"
+  disable: "deactivate, deselect, hide, turn off"
+  disabled: "deactivated, deselected, hidden, turned off, unavailable"
+  disables: "deactivates, deselects, hides, turns off, makes unavailable"
+  dummy data: "placeholder data"
   duplicate: "copy"
+  easy: "efficient"
+  easily: "efficiently"
   execute: "run, start"
+  first-class: "top-level"
+  flesh-colored: "dark brown, cream, beige"
   hack: "tip, workaround, configure, modify"
   hit: "select, press, visits"
-  invalid: "not valid, incorrect"
```

Comment:
I think we should keep this here since `invalid` can be seen as offensive. I would omit `not valid` since that's a little wordy, IMO.